### ISSUE TITLE
Support Scope(Spans|Metrics) before June 15, 2022

### ISSUE THREAD

**Issue by robbkidd** [honeycombio/husky#70]
`InstrumentationLibrary*` fields are [scheduled to be removed from proto June 22, 2022](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/common/v1/common.proto#L72). 

[proto 0.15.0](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v0.15.0) is the version that `Scope*` fields were introduced and `InstrumentationLibrary*` fields were deprecated (but still present). That is the first version that supports incoming OTLP with either prefix.

**Comment by MikeGoldsmith** [comment]
I don't think we don't need to update by June 15th. The change to introduce Scope(Spans|Metrics) put them in the old position of Instrumentation(Spans|Metrics), so existing receivers will continue to receive Instrumentation(Spans|Metrics) data.

The issue we saw with otel-python was down to them upgrading the proto that introduced Scope(Spans|Metrics) but continued to use the older Instrumentation(Spans|Metrics) placeholder, which meant they were added to a different location in the protobuf packet. This was a bug and has since been fixed. The expectation was that any sender upgrading to a newer version of the proto definitions that included Scope(Spans|Metrics), would update to use the newer structures.

The result of removing Instrumentation(Scope|Metrics) in a future proto definition release would only remove the option for a sender to set the wrong thing.

However, saying all that - we are currently locked into v0.11.0 of the proto spec and that does need to change soon.
